test_wazuh/test_fim/test_ambiguous_confs/test_ambiguous_simple.py

```diff
@@ -131,21 +157,28 @@ def no_report_changes_validator(event):
         """
         for file in file_list:
             diff_file = os.path.join(WAZUH_PATH, 'queue', 'diff', 'local')
+            print(f'1{diff_file}')
             if sys.platform == 'win32':
                 diff_file = os.path.join(diff_file, 'c')
-                diff_file = os.path.join(diff_file, folder.strip('C:\\'), file)
+                diff_file = os.path.join(diff_file, folder.strip('c:\\'), file)
             else:
                 diff_file = os.path.join(diff_file, folder.strip('/'), file)
             assert not os.path.exists(diff_file), f'{diff_file} exists'
             assert 'content_changes' not in event['data'].keys(), f"'content_changes' in event"
 
+    check_apply_test(tags_to_apply, get_configuration['tags'])
     scheduled = get_configuration['metadata']['fim_mode'] == 'scheduled'
+    file_list = ['regular']
+    min_timeout = DEFAULT_TIMEOUT
```

Comment:
We already import DEFAULT_TIMEOUT as a constant, there is no need to assign it to a variable